mobile/uplink.go

```diff
@@ -146,7 +152,7 @@ func (project *Project) ListBuckets(cursor string, limit int) (*BucketList, erro
 	scope := project.scope.child()
 	opts := libuplink.BucketListOptions{
 		Cursor:    cursor,
-		Direction: storj.Forward,
+		Direction: storj.After,
```

Comment:
maybe let's change the `cursor` name here, to make it clear that it defaults to after. e.g. `after string`